plugins/default/lib/test_component_health.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import unittest
+
+from component_health import parse_response
+
+class TestParseResponse(unittest.TestCase):
+    def runTest(self):
+        self.assertEqual(parse_response("{\"status\": \"ok\", \"summary\": \"Test Summary\", \"details\": []}"), (0, 'Test Summary', []))
+        self.assertEqual(parse_response("{\"status\": \"warn\", \"summary\": \"Test Summary\", \"details\": []}"), (1, 'Test Summary', []))
+        self.assertEqual(parse_response("{\"status\": \"crit\", \"summary\": \"Test Summary\", \"details\": []}"), (2, 'Test Summary', []))
```

Comment:
Maybe instead of the magic 0, 1, 2, here we really want to use the nagios constants?